### PR TITLE
docs: created the first documentation around Resizetizer

### PR DESCRIPTION
**Pull request by pictos**
This is a draft PR with the content of the docs for early review.

The content will be update

### CODE REVIEW

File: docs/Resizetizer_Get_Started.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+> [!TIP]
+> Feel free to delete the old assets related to app icon in the project.
+
+## UnoSplashScreen
+
+`UnoSplashScreen` is the build action for the splash screen. There should only be one per application. The `UnoSplashScreen` has two more properties that you can use to adjust your asset, they are:
+
+* BaseSize: It's the size that will be used to perform the scaling of the image. The default value is the size of asset. So if you feel that your SplashScreen doesn't look right you can tweak this value.
+
+* Color: It's the background color of that will be used to fill the empty space on the final SplashScreen asset. The default value is `#FFFFFF`(transparent).
```
Comment (jeromelaban) [inline docs/Resizetizer_Get_Started.md:133]: Let's add a code sample for these two properties.

File: docs/Resizetizer_Get_Started.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### 5. Configuring the project to use generated splash screen
```
Comment (jeromelaban) [inline docs/Resizetizer_Get_Started.md:135]: Are you missing build pieces to finish this part?

File: doc/using-uno-resizetizer.md [hunk 3]
```diff
@@ -1,4 +1,211 @@
...
-## Getting started with Uno.Resizetizer
\ No newline at end of file
+
+# How-To: Get Started with Uno.Resizetizer
+
+Uno.Resizetizer is a set of MSBuild tasks to manage the assets of an app. Using this package, it is not needed to care about generating and maintaining different image sizes/scaling and create a splash screen plumbing. It is only needed to provide an `svg` file and the tool will take care of the rest.
+
+This tutorial will walk through how to use it on your Uno Platform app. To create an app, make sure to visit [our getting started tutorials](xref:Uno.GetStarted).
+
+## Installation
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:11]: All things considered, as we are including resizetizer in the default template, we could say that the following installation steps are already done when using Uno 4.8.

```suggestion
Starting Uno Platform 4.8, the solution template includes Uno.Resizetizer by default. If you want to use it with an application created from an earlier version of Uno Platform, you can follow the steps below.

> [!TIP]
> To create an app, make sure to visit [our getting started tutorials](xref:Uno.GetStarted).

## Manual Installation
```

File: doc/using-uno-resizetizer.md [hunk 4]
```diff
@@ -1,4 +1,211 @@
...
+
+```xml
+<ItemGroup>
+    <UnoIcon Include="$(MSBuildThisFileDirectory)Icons\iconapp.svg"
+             ForegroundFile="$(MSBuildThisFileDirectory)Icons\appconfig.svg"
+             Color="#FF0000"/>
+</ItemGroup>
+```
+
+Next, some adjustments are needed on `Android`, `Windows`, `wasm`, `mac-catalyst` and `iOS`. Let's start with `Android`.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:82]:
```suggestion
Next, some adjustments are needed on `Android`, `Windows`, `WebAssembly`, `mac-catalyst` and `iOS`. Let's start with `Android`.
```

File: doc/using-uno-resizetizer.md [hunk 5]
```diff
@@ -1,4 +1,211 @@
...
+       DisplayName="Resizetizer.Extensions.Sample"
+       Description="Resizetizer.Extensions.Sample">
+       <uap:SplashScreen Image="Resizetizer.Extensions.Sample/Assets/SplashScreen.png" />
+       <uap:DefaultTile/>
+     </uap:VisualElements>
+   </Application>
+ </Applications>
+```
+
+Now let's jump to the wasm platform.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:117]:
```suggestion
Now let's jump to the WebAssembly platform.
```

File: doc/using-uno-resizetizer.md [hunk 6]
```diff
@@ -1,4 +1,211 @@
...
+
+> [!NOTE]
+> You just need to configure it, if you want to ship it as PWA.
+
+* Opens the `manifest.webmanifest` file and look for the `icons` tag, now remove all the values inside it. And leave an empty array like this:
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:122]:
```suggestion
* Open the `manifest.webmanifest` file and look for the `icons` tag and remove all the values inside it. You should be seeing an empty array like this:
```

File: doc/using-uno-resizetizer.md [hunk 7]
```diff
@@ -1,4 +1,211 @@
...
+       <uap:DefaultTile/>
+     </uap:VisualElements>
+   </Application>
+ </Applications>
+```
+
+Now let's jump to the wasm platform.
+
+> [!NOTE]
+> You just need to configure it, if you want to ship it as PWA.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:120]:
```suggestion
> You will only need to configure this platform if you want to deploy it as a PWA.
```

File: doc/using-uno-resizetizer.md [hunk 8]
```diff
@@ -1,4 +1,211 @@
...
+```json
+{
+  "background_color": "#ffffff",
+  "description": "UnoResApp1",
+  "display": "standalone",
+  "icons": [ ],
+}
+```
+
+Now let's jump to the Apple's platform.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:133]:
```suggestion
Now let's jump to Apple platforms.
```

File: doc/using-uno-resizetizer.md [hunk 9]
```diff
@@ -1,4 +1,211 @@
...
+
+> [!NOTE]
+> If your app icon has other name than `iconapp` use it instead.
+
+> [!TIP]
+> Feel free to delete the old assets related to app icon in the project.
+
+## UnoSplashScreen
+
+`UnoSplashScreen` is the build action for the splash screen. There should only be one per application. The `UnoSplashScreen` has two more properties that you can use to adjust your asset, they are:
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:145]:
```suggestion
`UnoSplashScreen` is the build action for the splash screen. There should only be one per application. The `UnoSplashScreen` has two more properties that you can use to adjust your asset, which are:
```

File: doc/using-uno-resizetizer.md [hunk 10]
```diff
@@ -1,4 +1,211 @@
...
+
+* BaseSize: It's the size that will be used to perform the scaling of the image. The default value is the size of asset. So if you feel that your SplashScreen doesn't look right you can tweak this value.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:147]:
```suggestion
* `BaseSize`: It's the size that will be used to perform the scaling of the image. The default value is the size of asset. So if you feel that your SplashScreen doesn't look right you can tweak this value.
```

File: doc/using-uno-resizetizer.md [hunk 11]
```diff
@@ -1,4 +1,211 @@
...
+
+* Color: It's the background color of that will be used to fill the empty space on the final SplashScreen asset. The default value is `#FFFFFF`(transparent).
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:149]:
```suggestion
* `Color`: It's the background color of that will be used to fill the empty space on the final SplashScreen asset. The default value is `#FFFFFF`(transparent).
```

File: doc/using-uno-resizetizer.md (hunk 3 above)
Comment (pictos) [reply doc/using-uno-resizetizer.md:11]: @jeromelaban even the `blank` template will have Resizetizer? Just trying to cover all scenarios here

File: doc/using-uno-resizetizer.md [hunk 12]
```diff
@@ -1,4 +1,211 @@
...
+```xml
+<UnoSplashScreen
+         Include="$(MSBuildThisFileDirectory)Splash\splash_screen.svg"
+         BaseSize="128,128"
+         Color="#512BD4" />
+```
+
+Next some adjustments are needed on `Android`, `Windows`, and `iOS`. Let's start with `Android`.
+
+* Opens the `Style.xml` file, look the `Theme` that is been used by the application and add the following line:
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:165]:
```suggestion
* Open the `Style.xml` file, look for the `Theme` that is been used by the application and add the following line:
```

File: doc/using-uno-resizetizer.md [hunk 13]
```diff
@@ -1,4 +1,211 @@
...
+       <uap:SplashScreen />
+       <uap:DefaultTile />
+     </uap:VisualElements>
+   </Application>
+ </Applications>
+```
+
+Now let's jump to iOS platform.
+
+* Opens the `info.plist` file and look for the `UILaunchStoryboardName` key, delete it and its value.
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:205]:
```suggestion
* Open the `info.plist` file and look for the `UILaunchStoryboardName` key, delete it and its value.
```

File: doc/using-uno-resizetizer.md [hunk 14]
```diff
@@ -1,4 +1,211 @@
...
+    <item name="android:windowSplashScreenBrandingImage">@drawable/uno_splash_image</item>
+</style>
+```
+
+> [!NOTE]
+> The `uno_splash_image` and `uno_splash_color` are generated by the build process.
+
+Now let's jump to Windows platform.
+
+* Opens the `Package.appxmanifest` file and look for the `Application` tag, inside that look for the `uap:SplashScreen` tag. Delete the `Image` property and its value, the `Application` tag should like this:
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:186]:
```suggestion
* Open the `Package.appxmanifest` file and look for the `Application` node, inside it look for the `uap:SplashScreen` node. Delete the `Image` property and its value, the `Application` tag should like this:
```

**Review by jeromelaban — APPROVED** [review]

File: doc/using-uno-resizetizer.md [hunk 15]
```diff
@@ -1,4 +1,211 @@
...
+* Create a `Icon` folder inside the Base project, and add the files related to app icon there.
+* Now open the `base.props` file, inside the `MyApp.Base` folder project and add the following block
+
+```xml
+<ItemGroup>
+    <UnoIcon Include="$(MSBuildThisFileDirectory)Icons\iconapp.svg"
+             ForegroundFile="$(MSBuildThisFileDirectory)Icons\appconfig.svg"
+             Color="#FF0000"/>
+</ItemGroup>
+```
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:80]: You'll need to remove the `\` in front of the code blocks, github does not like it much :)

```suggestion
# [**Class Library Based Solution**](#tab/classlib)

* Create a `Icon` folder inside the Base project, and add the files related to app icon there.
* Now open the `base.props` file, inside the `MyApp.Base` folder project and add the following block:

```xml
<ItemGroup>
    <UnoIcon Include="$(MSBuildThisFileDirectory)Icons\iconapp.svg"
             ForegroundFile="$(MSBuildThisFileDirectory)Icons\appconfig.svg"
             Color="#FF0000"/>
</ItemGroup>
\```

# [**Shared Project Based Solution**](#tab/sharedproject)

* Create a `Icon` folder inside the Shared project, and add the files related to app icon there.
* In each of your project heads (iOS, Android, WebAssembly, ...) add the following block:

```xml
<ItemGroup>
    <UnoIcon Include="..\MyApp.Shared\Icons\iconapp.svg"
             ForegroundFile="..\MyApp.Shared\Icons\appconfig.svg"
             Color="#FF0000"/>
</ItemGroup>
\```

***
```

File: doc/using-uno-resizetizer.md [hunk 16]
```diff
@@ -1,4 +1,211 @@
...
+
+* Create a `SplashScreen` folder inside the Base project, and add the file related to splash screen there.
+* Now open the `base.props` file, inside the `MyApp.Base` folder project and add the following block
+
+```xml
+<UnoSplashScreen
+         Include="$(MSBuildThisFileDirectory)Splash\splash_screen.svg"
+         BaseSize="128,128"
+         Color="#512BD4" />
+```
```
Comment (jeromelaban) [inline doc/using-uno-resizetizer.md:161]: Make sure to remove the `\` in from the end of the block

```suggestion
# [**Class Library Based Solution**](#tab/classlib)

* Create a `SplashScreen` folder inside the Base project, and add the file related to splash screen there.
* Now open the `base.props` file, inside the `MyApp.Base` folder project and add the following block

```xml
<UnoSplashScreen
         Include="$(MSBuildThisFileDirectory)Splash\splash_screen.svg"
         BaseSize="128,128"
         Color="#512BD4" />
\```

# [**Shared Project Based Solution**](#tab/sharedproject)

* Create a `SplashScreen` folder inside the Shared project, and add the file related to splash screen there.
* In each of your project heads (iOS, Android, WebAssembly, ...) add the following block:
```xml
<UnoSplashScreen
         Include="..\MyApp.Shared\Splash\splash_screen.svg"
         BaseSize="128,128"
         Color="#512BD4" />
\```

```

File: doc/using-uno-resizetizer.md (hunk 3 above)
Comment (jeromelaban) [reply doc/using-uno-resizetizer.md:11]: I think it should be having it, yes.

File: doc/using-uno-resizetizer.md (hunk 16 above)
Comment (pictos) [reply doc/using-uno-resizetizer.md:164]: @jeromelaban for `Legacy` we add the `UnoSplashScreen` and `UnoIcon` on the `.projitems`, don't you think it's better? You can see it [here](https://github.com/unoplatform/uno.resizetizer/blob/8e7fa13303f580a8b9a8614a26165100ebf4d06d/samples/Legacy/Uno.Resizetizer.Sample/Uno.Resizetizer.Sample.Shared/Uno.Resizetizer.Sample.Shared.projitems#L48)

**Pull request merged**